inbm-lib/inbm_common_lib/shell_runner.py

```diff
@@ -7,12 +7,15 @@
 import platform
 import os
 import shlex
-import subprocess
-import sys
+
 import logging
 import builtins
 from datetime import datetime
-from subprocess import Popen, PIPE
+
+# We run the process with shell=False, This class is used to run 
+# what has to be ran with Popen securely or with low risk
+from subprocess import Popen, PIPE # nosec: B404
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** Avoid using shell=False comment as it is not necessary and does not add value to the security of the code.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
from subprocess import Popen, PIPE
```